scripts/legislation/africa/south_africa_english.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^I. In this Act/,$p'
+}
+
+function prefix_article_numbers_with_article_literal {
+  sed -E 's/^([0-9]+\.)/Article \1/'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function move_article_titles_after_article_numbers {
+  sed -En '${p;q};N;/\nArticle/{s/^(.*)\n(Article [0-9]+\.) /\2 \1|/p;b};P;D'
+}
+
+function move_article_titles_above_articles {
+  sed -E 's/^(\([0-9]+\) )([^|]+)\|/\2\n\1/'
+}
+
+function remove_line_numbers {
+  sed -E 's/ 5 | 10 | 15 | 20 | 25 | 30 | 35 | 40 | 45 | 50 | 55 | 60/ /g' 
+}
+
+function amend_errors_in_articles {
+  sed -E 's/-- /: /g' | \
+    sed -E 's/- /: /g' | \
+    sed -E 's/ \(([0-9]+)\)/ [\1]/g' | \
+    sed -E 's/([0-9]+) (\[[0-9]+\])/\1\2/g' | \
+    sed -E 's/ \(a\)/ \[a\]/g' | \
+    sed -E 's/subsection \(([0-9]+)\)/subsection [\1]/g' | \
```

Comment:
Isn't the `replace_parentheses_around_article_delimiters_with_square_brackets` function supposed to do this?